DuckDuckGo/NetworkProtectionDebugViewController.swift

```diff
@@ -63,6 +66,12 @@ final class NetworkProtectionDebugViewController: UITableViewController {
 
     }
 
+    enum RegistrationKeyRows: Int, CaseIterable {
+
+        case expireNow
```

Comment:
In the future we could add last rekey date here, so it's easy to see how well this is working in real-world scenarios, but we don't need to do it here. We have a project dedicated to the debug menu which tbh I would like to break up into tasks, so it can be done as a part of that.